src/ContentManager/ContentManager.php

```diff
@@ -117,15 +117,13 @@ private function populateDataTypeSeederContent(string $stub, DataType $dataType)
         $stub      = $this->populatePermissionStatements($stub, $dataType);
         $stub      = $this->populateMenuStatements($stub, $dataType);
 
-        list($dataType, $stub) = $this->populateTranslationStatements($stub, $dataType);
+        [$dataType, $stub] = $this->populateTranslationStatements($stub, $dataType);
```

Comment:
Will possibly not work on Laravel 5.4 and 5.5 as array destructuring is only available since PHP7.1

Reply:
This change was reported by StyleCI so I made this change. I think we will need to make change on styleCI configuration on dashboard.

Reply:
@samundra better to revert this change in this PR and fix CI issue in another one

Reply:
Sounds good. I will revert this change.